editor/editor_node.cpp

```diff
@@ -4836,16 +4832,6 @@ Button *EditorNode::add_bottom_panel_item(String p_text, Control *p_item) {
 	return tb;
 }
 
-bool EditorNode::are_bottom_panels_hidden() const {
```

Comment:
For the reference, it seems this is still used in `3.2`, but was indeed removed in `master`:
```
editor/script_editor_debugger.cpp
850:                            if (EditorNode::get_singleton()->are_bottom_panels_hidden()) {
```